src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/taglets/LinkTaglet.java

```diff
@@ -30,30 +30,17 @@
 import javax.lang.model.element.Element;
 
 import com.sun.source.doctree.DocTree;
-import com.sun.source.doctree.LiteralTree;
+
 import jdk.javadoc.doclet.Taglet.Location;
+import jdk.javadoc.internal.doclets.toolkit.BaseConfiguration;
 import jdk.javadoc.internal.doclets.toolkit.Content;
 
-/**
- * An inline taglet used to denote literal code fragments.
- * The enclosed text is interpreted as not containing HTML markup or
- * nested javadoc tags, and is rendered in a font suitable for code.
- *
- * <p> The tag {@code {@code ...}} is equivalent to
- * {@code <code>{@literal ...}</code>}.
- * For example, the text:
- * <blockquote>  The type {@code {@code List<P>}}  </blockquote>
- * displays as:
- * <blockquote>  The type {@code List<P>}  </blockquote>
- */
-public class CodeTaglet extends BaseTaglet {
+public abstract class LinkTaglet extends BaseTaglet {
```

Comment:
`{@link}` has finally got its own taglet. However, I note that `{@code}` has lost its taglet.

In the markup sense, `{@link}` is to `{@code}` as `{@linkplain}` is to `{@literal}`. If you agree with this, then I think we should leave `{@code}`, rather than `{@literal}`.

Reply:
I agree, but may do that separately, or as part of subsequent cleanup

Reply:
I may leave the types as-is but try and fix it in the doc comments.   The problem is in the class names and the crossed-over functionality.